e2e/support/test_support.go

```diff
@@ -128,9 +128,9 @@ const ExpectedOSClusterRoles = 1
 
 var TestDefaultNamespace = "default"
 
-var TestTimeoutShort = 1 * time.Minute
-var TestTimeoutMedium = 5 * time.Minute
-var TestTimeoutLong = 15 * time.Minute
+var TestTimeoutShort = 5 * time.Minute
```

Comment:
I think these values are too high. If we need to change them specifically for certain tests, then, we should change the test and include the specific timeout instead.

Reply:
According to my empirical tests there are the one that minimize test runs failing due to small perturbations in load or network speed. Honestly I don't see may cons in bumping them, the only one is that actually failing tests will take longer. But that is a condition that should happens rarely since most of the runs should be of passing tests.

Reply:
That's exactly the problem. In case of an unstable change we may introduce, the time to complete a test would be at least duplicated, having to wait for a feedback too much time. Imho, the goal should be to have faster timeouts. If some test is not stable, then, I don't think the culprit is the timeout. And if it's the timeout, likely it's in the specific flaky test.

Reply:
I see both of you having a point here. IMO if the increased timeout really fixes the flaky tests I tend to accept the fact that we need to wait some more time when a test really is broken by the changes in a PR. The feedback loop for a successful test is over 60 minutes anyways and having a flaky test can be really exhausting.